URBNSwiftAlert/Classes/URBNSwAlertStyler.swift

```diff
@@ -0,0 +1,420 @@
+//
+//  URBNSwAlertStyler.swift
+//  Pods
+//
+//  Created by Kevin Taniguchi on 5/22/17.
+//
+//
+
+import Foundation
+import URBNConvenience
+
+public struct URBNSwAlertStyler {
+    
+    /**
+     * Pass no to disable blurring in the background
+     */
+    public var blurEnabled = true
+    
+    /**
+     * Background color of alert view
+     */
+    public var backgroundColor = UIColor.white
+    
+    /**
+     * Tint color of the view behind the Alert. Blur must be disabled
+     */
+    public var backgroundTintColor: UIColor?
+    
+    /**
+     * Radius of the blurred snapshot
+     */
+    public var blurRadius: CGFloat = 5.0
+    
+    /**
+     * Opacity of the alert view's shadow
+     */
+    public var alertViewShadowOpacity: Float = 0.0
+    
+    /**
+     * Radius of the alert view's shadow
+     */
+    public var alertViewShadowRadius: CGFloat = 0.0
+    
+    /**
+     * Color of the alert view's shadow
+     */
+    public var alertViewShadowColor = UIColor.clear
+    
+    /**
+     * Offset of the alert view's shadow
+     */
+    public var alertShadowOffset = CGSize.zero
+    
+    /**
+     * Tint color of the blurred snapshot
+     */
+    public var blurTintColor: UIColor = UIColor.white.withAlphaComponent(0.4) {
+        didSet {
+            assert(blurTintColor.cgColor.alpha < 1.0, "URBNAlertStyle: blurTintColor alpha component must be less than 1.0 to see the blur effect. Please use colorWithAlphaComponent: when setting a custom blurTintColor, for example: UIColor.white.withAlphaComponent(0.4)")
+        }
+    }
+    
+    /**
+     * Saturation blur factor of the blurred snapshot. 1 is normal. < 1 removes color, > 1 adds color
+     */
+    public var saturationDelta: CGFloat = 1.0
+    
+    /**
+     * Bool for using an animation to present alert view
+     */
+    public var isAnimated: Bool = true
+    
+    /**
+     * Duration of the presenting and dismissing of the alert view
+     */
+    public var animationDuration: CGFloat = 0.6
+    
+    /**
+     * Spring damping for the presenting and dismissing of the alert view
+     */
+    public var animationDamping: CGFloat = 0.6
+    
+    /**
+     * Spring initial velocity for the presenting and dismissing of the alert view
+     */
+    public var animationInitialVelocity: CGFloat = -10.0
+    
+    /**
+     * Margins on left and right sides of the alert, which sets the width constraint.  Used if no insets are given.  The alert view will use its most compressed layout values for height
+     */
+    public var horizontalMargin: CGFloat = 45.0 {
+        didSet {
+            assert(horizontalMargin > 0, "Margin must be greater than 0")
+        }
+    }
+    
+    /**
+     * Insets of the alert from the top / left / bottom / right of the screen.
+     */
+    public var alertWrappingInsets: UIEdgeInsets?
```

Comment:
I think you added this to take place of `alertMaxWidth` and `alertMinWidth`? I am tying to think of a reason why we would need to keep those around. I don't think we have used them much--- other than trying to have the alert inset a certain amount from the screen edges, and this property handles that.

I tried to set these to high values for the standard alert and got some funky layout. The message laid under the title, and the Done button was under the message field making it un-touchable
`oneBtnAlert.alertStyler.alertWrappingInsets = UIEdgeInsets(top: 100, left: 100, bottom: 100, right: 100)`. This may be a case where the developer has to just not use dumb values to avoid layout issues